docs/conf.py

```diff
@@ -19,7 +19,7 @@
 import os
 import sys
 
-sys.path.insert(0, os.path.abspath('..'))
+sys.path.insert(0, os.path.abspath('../src'))
```

Comment:
@kratsg Good now?

Reply:
Looks like it. I pulled this into a fresh Docker container, installed `master`, switched here, built the docs and seems the schema is in good shape too

```
~/data/pyhf/docs$ find _build/html/ -iname "defs.json"
_build/html/schemas/1.0.0/defs.json
```

Thanks very much for catching this in the first place!